DOCS/SETUP.md

```
@@ -35,52 +35,49 @@ npx expo start --web    # Web browser
 ### Environment Configuration
 
 Create a `.env` file (or set via Supabase dashboard):
+
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)